src/p5.dance.js

```diff
@@ -183,6 +184,28 @@ module.exports = class DanceParty {
     this.allSpritesLoaded = true;
   }
 
+  getUserCode(userCode) {
```

Comment:
I'd expect a "get" function to return a value.  This seems more like a "retrieve" or an "init" maybe?

Reply:
Oh yeah - had already updated this. Thanks!